composer.json

```diff
@@ -24,41 +24,12 @@
   ],
   "require": {
     "php": ">=5.3.3",
-    "symfony/symfony": "2.6.*",
-    "doctrine/orm": ">2.4,<2.6",
-    "doctrine/dbal": ">2.4,<2.6",
-    "doctrine/doctrine-bundle": "~1.2",
-    "symfony/assetic-bundle": "~2.3",
-    "symfony/monolog-bundle": "~2.4",
-    "sensio/distribution-bundle": "~3.0,>=3.0.12",
-    "sensio/framework-extra-bundle": "~3.0,>=3.0.2",
-    "incenteev/composer-parameter-handler": "~2.0"
+    "doctrine/dbal": "~2.4"
+  },
+  "suggest": {
+    "doctrine/orm": "To use DQL functions"
   },
   "require-dev": {
-    "sensio/generator-bundle": "~2.3",
     "phpunit/phpunit": ">4.2"
```

Comment:
the constraint ``>4.2`` does not have an upper bound, which is a bad practice (even though it is less an issue in dev requirements as these impact only your own packages, not people using it).
See https://getcomposer.org/doc/faqs/why-are-unbound-version-constraints-a-bad-idea.md for the reason why unbound constraints are a bad idea